tests/detect-bytemath-09/test.yaml

```diff
@@ -0,0 +1,13 @@
+requires:
+  min-version: 8
```

Comment:
Only the shell check requires min-version 8, right ?

Reply:
Yes, correct.